synfig-studio/src/gui/dialogs/vectorizersettings.cpp

```diff
@@ -270,8 +264,8 @@ void
 VectorizerSettings::set_progress(float value)
```

Comment:
It looks like this function is never used.

Reply:
I missed this one thanks for pointing :)